test/test_functionalization.py

```diff
@@ -3,9 +3,9 @@
 import torch
 from torch.testing._internal.common_utils import TestCase, run_tests, skipIfTorchDynamo, TEST_WITH_TORCHDYNAMO
 from torch.testing._internal.logging_tensor import LoggingTensor, capture_logs
-from torch.utils._pytree import tree_map
 from torch.fx.experimental.proxy_tensor import make_fx
 from torch.fx.passes.reinplace import reinplace
+from functorch.experimental import functionalize
```

Comment:
since the build systems are not unified yet, ideally you should test if functorch is actually importable

Reply:
oh, looks like CI is actually failing for that reason....

Any idea on the timeline for unifying build systems? Otherwise I guess I could try to import `functionalize()`, and fall back to the roll-your-own version that was in this file.

Reply:
to test in CI, just update the CI to start building functorch after regular pytorch

@zou3519 knows unified build timeline

Reply:
A short fix is to install functorch in all CI configurations so that it's available for testing.

Unified build timeline: by 1.13, unless people have reasons for it earlier